<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>780</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Archivo FITS</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dimensiones</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDimensionesArchivo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelNombreArchivo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Canales</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Primer canal (Angstrom)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resolucion (Angstrom)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="PrimerCanal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Resolucion">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabPrincipal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1281</width>
      <height>651</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="TabVisualizador">
     <attribute name="title">
      <string>Visualizador del cubo</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>109</y>
        <width>511</width>
        <height>511</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="MatplotlibWidget11" name="visualizacion_cubo" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>491</width>
         <height>491</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 255, 0);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>240</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="limite_cubo">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>55</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="umbral_cubo">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Umbral</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Límite</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="vis_canal">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>5</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QRadioButton" name="vista_modo_canal">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Canal</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>55</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Límite</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="limite_cubo_suma">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>55</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="vista_modo_suma">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>5</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Suma</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Umbral</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="umbral_cubo_suma">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>20</width>
         <height>90</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>10</y>
        <width>121</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_val_max">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Max.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_val_min">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Min.</string>
       </property>
      </widget>
      <widget class="QComboBox" name="colormap_cubo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentText">
        <string>Viridis</string>
       </property>
       <item>
        <property name="text">
         <string>Viridis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Grices</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Grices R</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12C - 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12C - 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cividis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plasma</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>741</width>
        <height>611</height>
       </rect>
      </property>
      <property name="title">
       <string>Perfiles</string>
      </property>
      <widget class="MatplotlibWidget33_visual" name="vis_perfil" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>581</width>
         <height>581</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>20</y>
         <width>91</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Pixel</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>17</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="pixel_x_vis">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>17</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>43</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="pixel_y_vis">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>43</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="boton_analizar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Analizar</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>369</x>
        <y>10</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="modo_click">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Por click</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Por click: Visualiza los perfiles haciendo click sobre el pixel.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Modo libre: Visualiza los perfiles pasando sobre ellos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="modo_libre">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Libre</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="TabPerfiles">
     <attribute name="title">
      <string>Analisis del perfil</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>91</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Pixel</string>
      </property>
      <widget class="QPushButton" name="boton_CargarPixel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>67</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Cargar pixel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>17</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="pixel_x">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>17</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>43</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="pixel_y">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>43</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>102</y>
        <width>521</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="MatplotlibWidget11" name="analisis_perfil" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>500</width>
         <height>500</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Página</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>741</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(milky_way.png);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget11</class>
   <extends>QWidget</extends>
   <header>clases_mpl_pyqt5</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MatplotlibWidget33_visual</class>
   <extends>QWidget</extends>
   <header>clases_mpl_pyqt5</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabPrincipal</tabstop>
  <tabstop>PrimerCanal</tabstop>
  <tabstop>vista_modo_canal</tabstop>
  <tabstop>Resolucion</tabstop>
  <tabstop>pixel_y</tabstop>
  <tabstop>limite_cubo</tabstop>
  <tabstop>boton_CargarPixel</tabstop>
  <tabstop>umbral_cubo</tabstop>
  <tabstop>colormap_cubo</tabstop>
  <tabstop>vis_canal</tabstop>
  <tabstop>pixel_x</tabstop>
  <tabstop>vista_modo_suma</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
